### PR TITLE
Add missing override annotation

### PR DESCRIPTION
**Pull request by xichen01**
### Why are the changes needed?
- Add missing @override annotation
- Removed duplicate comments that already existed in the base class after adding @override annotation

### Does this PR introduce any user facing changes?
Please list the user-facing changes introduced by your change, including
No

### CODE REVIEW

**Review by maobaolong — APPROVED** [review]
It LGTM, thanks for clear the code.

**Comment by maobaolong** [comment]
@jiacheliu3 @HelloHorizon Would you like to take another look?

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 1]
```diff
@@ -3474,14 +3474,7 @@ private void scheduleAsyncPersistenceInternal(LockedInodePath inodePath,
...
-   *
-   * @param path the path to sync
-   * @param changedFiles collection of files that are changed under the path to sync, if this is
-   *        null, force sync the entire directory
-   * @param executorService executor to execute the parallel incremental sync
-   */
+  @Override
   public void activeSyncMetadata(AlluxioURI path, Collection<AlluxioURI> changedFiles,
       ExecutorService executorService) throws IOException {
     if (changedFiles == null) {
```
Comment (dbw9580) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:3480]: Please preserve the comment on the behavior when `changedFiles` is `null`. Also please add a `@Nullable` annotation on this parameter.

File: stress/common/src/main/java/alluxio/stress/fuse/FuseIOTaskResult.java [hunk 2]
```diff
@@ -89,6 +89,7 @@ public void merge(FuseIOTaskResult result) {
   /**
    * @return the base parameters
    */
+  @Override
```
Comment (dbw9580) [inline stress/common/src/main/java/alluxio/stress/fuse/FuseIOTaskResult.java:92]: Remove these javadoc.

**Review by dbw9580 — CHANGES_REQUESTED** [review]
A few minor issues. Thanks!

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java (hunk 1 above)
Comment (dbw9580) [reply core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java:3480]: Actually this is more complicated: when `changedFiles` is not `null` but an empty collection, this method does nothing. It starts a full sync when `changedFiles` is `null`. Should update the comments to document this behavior.

**Review by dbw9580 — APPROVED** [review]
LGTM, thanks!

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**